Tests iOS/AddTagsItemTests.swift

```diff
@@ -55,10 +55,13 @@ class AddTagsItemTests: XCTestCase {
         let addTagsView = app.addTagsView.wait()
         addTagsView.wait()
         addTagsView.newTagTextField.tap()
-        addTagsView.newTagTextField.typeText("Tag 1")
+        addTagsView.newTagTextField.typeText(XCUIKeyboardKey.delete.rawValue)
```

Comment:
FYI - test still pass after deleting this line for me, so not sure if we need to have, but I'm okay with keeping it in as well!